and transcriptomics. So with me, we will talk about DNA and about RNA, of the yarn in general, and how they are converted in a formula stream of vectors and this kind of information, okay? So I'm supposed to give you as much as possible information in four hours. And it is the first time we do this course. It means that you are basically an experimenter. Is that OK for you? We'll see what we are able to reach, what kind of knowledge. Possibly the knowledge that I will transfer you will be much more than the knowledge that is required to pass this part of the exam and it's just to enrich your basic knowledge on this topic and just to give you some material that can be useful maybe possibly in the future and so on. So my next question is what is your background? I think I have a degree in informatics, I guess, maybe here. Okay. Any exam in the past concerning chemistry? No. Biology? No. Me? No, not really. It's a no. OK. OK. Any basic knowledge of chemistry and biology from high school?

Yeah. So we will go there, OK? And we'll reach there, OK? Feel free to interrupt me whenever you want. Feel free to tell me, even in a very direct way. No way. is not my language, okay? I will try to repeat it in a language more similar to you, okay? I say a little. It's an experiment. You will see that. I do a lot of experiments because I'm an experimental biologist. So you are my next experiment. Is that okay for you? Okay. So, it will be for you four hours of microbiology and four hours of biochemistry. Again, basic knowledge of biochemistry. And that will be done by my colleague, Gerardo Poli, who will come next week. We were asked, so these are emails. You can write to us whenever you want. If you have doubts, if you want other information, if you have fear that this exam will be very difficult, if you want. But you can contact my email, okay?

And we were asked, to tell you the reference books and that was the hardest question for us because the only books we have and we use are kind of fit like this and of course these are not fit for a course of 4 plus 4 hours, right? but still you have some means here because this is the reference of our knowledge, ok? so you have two books of microbiology, the first one is a very technical one So if you want to know how DNA sequencing is performed, you basically go here. And here is another book on molecular biology. And you have other two books, very technical and very basic knowledge of biochemistry. But what I really want to tell you is that we are going to give you slides, OK? already, the slides of today, loaded in our Google page. So what we will ask you will surely be present in the slides. So, it's just if you find that you didn't expect the Molecular Biology to be so beautiful for you, then you have some books if you want to read more. Okay, that is the idea.

Okay, sometimes you will find in the slides For example, a doi. Do you know what a doi is? I don't know how to tell you, but it's like the referment to an article in the literature or to a conference where something is being presented. Yes, exactly. It's a unique code that identifies a scientific paper of whatever origin. I like your answer. So in the degree course of biology, your age will be one more, typically. So I appreciate it. And yes, so sometimes we reference to some papers that you find in the literature. So if you go into Google and you write down this DOI, then the paper will appear and you can read it if you want. Okay? Okay. And other two very important points. The first concerns this. So we have here the slides, but the slides contain material that belongs to books. and then have authorized. So it's a kind of deal we make. I give the slides, but then we don't put the slides in public sources, because otherwise I won't be authorized to do that.

So we do it to help, especially this time. So we give you what we want, he authorizes you to do that. So he's a man of it. And then we talk about the exam. So Professor Milazzo has told you, sure. We will have a small written test concerning this part. And this written test will be performed possibly, so if you all like the idea, as an itinerary test, one week after the end of the lectures, up to the end of the eight hours of lectures, okay? So there will be the possibility to perform an exam in itinerary. So written test, half an hour, and That gives you the possibility to do the other part of the exam. So that was scheduled on Friday the 3rd of October, but of course if we now postpone one lecture, then the exam will be postponed too. So we have to see how it will be scheduled. Of course, if you don't feel like you're ready to do the exam on that date, you can always choose to do it in the official exam date, and that is also free.

Now, we recommend that you perform it now, because it's just the basic knowledge that allows you to understand better also the rest part of the exam. So, it's taught in this way, okay? So if you have questions, if you don't, where are you from? I'm from Germany. Germany, where? Dusseldorf. Dusseldorf, yeah, I know. I was in Bonn for one year during my studies there. So I know that. Okay, so let's start in a very boring way with the definitions, okay? Definition with the look from the biology perspective. So what do we mean by biologists? We mean by system biology. So the system biology is an analytical approach that is used to decipher the complexity of biological systems. So is that you have a problem and you try to watch it and you study it from a general perspective that takes account of all the parts making part of your system, okay? And it assumes that the network that these particles form together are much more informative than the very sum of these parts.

And so it's really worth to study it all together because you will get Of course this can be done only in multidisciplinary mode, so basically to do system biology you need many different disciplines, biology, theory, computer science, engineering, bioinformatics, physics and many more. of the holistic approach produced in this nice picture is this. You have a very complex biological system. You don't understand anything about it. But you are able to reduce this biological complexity. And that biological complexity is reduced because it is converted in a informative context. In a computational context, I know a very easy example about it. I have the human genome. Do you know what a genome is? Like the sequence of genes that you have? That I have in each of my cells. So my body is done by several cells. All the cells contain DNA. No, that's not really true. Red blood cells do not contain DNA. But all the other, yes, they do contain. And this DNA is done by several genes.

but we'll see that. But a genome is the entire sequence of DNA that is contained in each of my cells. How can I know it? Well, in a biological way, there's no way. It's too long. It's definitely too long. So we will see that the genome is organized in very, very long molecules. And I have no way experimentally to deduce the sequence of these molecules. It's too long for me. But if I fragment it, and this is what is done, in very small pieces, then that small piece, I am able to deduce the sequence because experimentally I am able because it's much shorter. So I obtain a lot of different pieces. that must be put in the correct order. But that is no more a biological problem, it's a computational problem. You get it? Sort of. Do you get it? So, other definition. What do we mean by omics? So omics is a suffix that is used in several is another possibility, and this suffix is derived from a Latin suffix "-ome", which means mass or many. So what does it mean basically?

It means that with that extreme you want to study a large number of data rather than one or a few. If I want to study the genome, I want to study the entire amount of genes of DNA that I have in my cells. If I want to study the proteome, then I want to know the entire amount of proteins that I have in my cell. Was it a rapid change in biology? Yes, definitely. When I started my thesis, it was yesterday almost, I'm talking about my master degree thesis. I was asked to study one protein. I loved it. It was my thesis protein. I produced it in the lab. I watched it in several different techniques. I put it in the cells. I saw the localization of this protein. I was so happy to know so many things about this protein. But this is one protein. And how many proteins do we have in our cells? It's 20, I don't know. 20 is something else. Yes, 20 is something else. But it's related. I don't remember what it is. We come to that.

So we don't have tens, we don't have hundreds, we don't have thousands. We have some tens of thousands, which we can write. So we have so many. So if I know everything about one protein, all the others, what about all the others, okay? So when I started to do molecular biology in the lab, I was asked to study one protein. 25 years later, here are sometimes a group of proteins, how they do change with treatments, with pathology, with development, and so on. So it was a radical shift So, when I started to do molecular biology in the lab, I was asked to study one protein. Twenty-five years later, here I am, one of the times, a full-blown protein. How they do change with treatments, with pathology, with development, and so on. So, it was a radical shift in our discipline. And this shift requires your expertise, because We need computational work to do it because it's something we can't handle in the lab studying tens of thousands of proteins together. Do you know how much time I need to produce one protein in the lab?

And another definition that we keep in mind, which is an important one, is primary sequence. So for many of the omics disciplines, I already mentioned genomics, proteomics, transcriptomics. The goal is that to find, to identify molecules DNA, RNA, proteins, whatever, by defining the primary sequence. What is the primary sequence? It's reported here. It is the exact specification of their atomic composition. So we are very lucky, you know, because these important molecules, the most important biomolecules that we have in our body, which are DNA, RNA, and proteins, Chemically, they are easy. They are linear polymers, okay? So, in nature, you have different types of polymers available. This is an example of linear polymer, this is an example of branched polymer, and this is an example of cross-linked polymer, okay? Luckily, very luckily for us, DNA, RNA proteins, are linear polygons.

What does it mean that they are formed by a linear bond of monometric units that are depicted here as balls that are just formed one after the other. So basically, the nature of the DNA molecule is determined by the nature of these monomers, which are also called building blocks. So which type of monomer is there? And of course, how many monomers do you have? So it's a matter of composition and of length. And that is what the theory is, the type of polymer that you have. And you can have a linear DNA molecule that is 50 monomers long, and another that is 50,000. And of course, the two molecules would be highly different. And this applies also for the proteins. Once the cell is synthesizing this molecule as a linear polymer, the linear polymer acquires a shape, a three-dimensional structure, that again depends on the primary sequence. So the type of monomers and the length of the chain. OK? OK. So these different structures are also referred to folds.

So when you write in the book protein folds, the structure of the anemone. The structure of the anemone refers to the three-dimensional shape that this molecule has, but this shape depends on the primary sequence, which is a string of monomeric units. Is that okay? Okay, so let's talk a little bit about the monomers again, these building blocks. So, I introduce you two types of building blocks. On one side you have nucleotides. So this is the chemical structure of a nucleotide. We have in our body four types of nucleotides. And instead we have 20 types of amino acids. So proteins have 20 possible building blocks. But these 20 billion blocks give rise to a lot of different molecules. Let's see how many are there. Just a general idea. So, let's start from the nucleotides. So, four possible nucleotides give rise to two possible linear polymers, which are called DNA and RNA. So, DNA DNA is organized in 46 molecules, ok?

So in our body, in each of our cells, besides the retrocytes, we have 46 molecules of DNA. These molecules are called chromosomes. It's something that we knew it, right? That we don't have chromosomes. We give some more information about it. For example, we know now, I tell you, that in these 46 molecules, there are around 3 per 10 power of 4 genes. So 30,000 genes. So they are linear chains, and these linear chains do contain millions of building blocks. Building blocks are called nucleotides. Okay? And if I sum together all the nucleotides belonging to the 46 prongions that we have in our cells, The total length that I have is 6 billions of nucleotides, which corresponds to a linear length of 2 meters. So in each of our cells, we have 2 meters of DNA. But cells do not contain only two liters of DNA. They contain also RNA. What about RNA? So each cell typically contains one liter of molecules of RNA. These molecules are a little bit shorter than the chromosomes.

So they can reach 20,000 nucleotides, but they can also be very, very short, down to 20 nucleotides. And there are several types of RNA that are doing stuff in our cells. RNA is a very fascinating molecule. It has recently become a drug, right? Possibly you have taken it. Yes, I'm sure, pretty sure. What are we talking about? Yeah. You got it? Wonderful drug. OK, and what about the 20 amino acids? About the 20 amino acids, we've got linear chains of proteins. And we do have a lot of proteins in our cells. Roughly some hundreds of thousands to millions of molecules. And again, the length can be up to 35,000. OK? All this stuff is all bound. In a very complex environment, it becomes a cell. So how many cells do we have in our body? What do you think? It's more. Maybe millions. I don't know. You don't know. You would say many. I would say millions. Yeah, yeah. OK. And it's more. So it's not easy to recover this information in the literature, right?

So 10 to the power of 13 cells in our body. I mean, could be, could be. We have several. I mean, one cell is, how long is a cell? You know this. So when we know, and it is very difficult problem, our answer depends. So in my logic, depends is a good answer for several things. And it depends, because there are some cells that are very small, and cells that are very long. Think of the neurons that you have in your spinal cord here. There are some that have the center of the sentence here, but the protrusion ends in the skin of your finger. So if you're a basketball player, then that cell can be meters long. And then all these cells get pumped in several tissues, which are categorized in four types of tissues. The tissues give rise to seven, eight organs that you have in your body. And all together, they form the living human organism.

So I like to start with this line because it gives you the idea of complexity and the need to monolistic approach, to understand all this kind of complexity. So now think about me. There are 25 years ago I was studying only one of these proteins. What for? No, it was a good study, I have to say. I had fun and I learned a lot of different things. I'm not saying that it's not useful anymore, this kind of approach, hypothesis-driven approach. I have a protein, that protein is involved in brain cancer, and I have to study it. Yes, I do have to study it. But still, there is no approach. To study all proteins and genes together is very powerful, and we have to exploit it, because now we have tools to do that. Okay, so with me, we are going to learn a little bit more about DNA analysis. Instead, with Professor Caruso, you will learn something more about proteins.

As I told you, the information that I will give you is a thought to give you some basic knowledge that can help you with the bioinformatics part of this course, okay? So, analysis of data sets from genomics and transcriptomics, for example.

Is that the part of the lesson that also is a part to help you in the system biology part, the course that you could do professionally. Is that okay? Okay. So let's start. Do all of you know that DNA has the shape of double helix? Yes. Okay. Here it is. It has definitely And in this twisted ladder, you can recognize the rails, which are depicted here in gray, and the rails and the ramps, okay, which are here depicted in different colors. So the ladder is twisted, so it is wrapped on itself. And this schematic picture represents the atomic composition of the DNA. What we learn in these two hours to four hours is that DNA is a wonderful molecule. And it is doubtful, because it contains a lot of information. that are put together and in this way it can be opened and copied. This process is called DNA regulation. So DNA is the molecule that stores genetic information. a redundant molecule. Because if you are redundant, then the storage is safer. You won't lose it. So basically, imagine DNA

as two molecules in one, which are the two So now imagine opening, and this is what happens in real education, each part can be used as So starting from one molecule of DNA, you open it, and you form two molecules of DNA. And this is something that, of course, in nature, any time, a cell has to divide. Because if a cell wants to divide, for example, during development, right? During development, one or few cells become many cells. And each time the cell divides, the cell must duplicate the DNA for division. is replicated in a very easy way. It's just unzipping the ladder, the helix, and using both parts to rebuild the molecule. And this very easy mechanism has been converted in a technology. And the sequencing technologies that to create the basis for genomics and transcriptomics all rely on the possibility that DNA is easily replicated. You just need building blocks and you need enzymes putting one building block after the other. This enzyme is called DNA polymerase. The other thing we will learn in these four hours is that DNA contains genes. So what are genes? DNA contains genes.

So genes are clusters of nucleotides, functional clusters of nucleotides that contain information. So genes are under genetic information. So the information stored in genes is very important because that information is used to build up proteins. So all the proteins that we have in our cells, the existence of these proteins rely on the fact that I have genes coding for those proteins, okay? It is called genetic code. So the genetic code is the code by which the information stored in genes becomes information to build up proteins. And finally, another thing we have to understand is that this is an easy thing. The DNA must be compacted. Why is that? Well, it is very easy to understand. I mean, the DNA that we have in each of our cells is two meters long. But most of my cells are not two meters long. I'm not a basketball player, right? I don't have only neurons, very normal neurons in my body. So most of the cells have a length of some kind of microns.

DNA is always two meters long, because all the cells have the same DNA. So it must be packed. Very, very tightly packed. But at this rate of compaction, the death means the possibility that the gene is expressed or not. So if the DNA is kind of more relaxed, then the gene information will become a protein. If the gene is very compacted, then the information will stay there and no protein will be produced. So if the gene is free, is not compacted, I have gene expression and I have a protein. If the gene is very compacted, I will not have gene expression and will not have any protein produced. That was a very long story made very short. So now we have to enter a little bit more in the detail. And we start with the building blocks, with the monomers. So the monomers of DNA and RNA are nucleotides. They are called nucleotides. And here is the chemical formula of nucleotides. Can you influence which genes are affected and which are not?

Can you influence which genes are affected and which are free? Can I, as a researcher, or I personally? Can the cells influence? Yeah, yeah. Yeah, that's a wonderful question. But a human brain is very different from a muscle on the leg. Why is that? We have the same DNA. Because the DNA in the function is very different. So the genes that are responsible for the function are not necessary for the brain. So the brain decides that it won't use them. But they want the genes that are important for synaptic functions, right? So what happens is that the genes of the contractions are compacted. And they don't use it. Why? Because they're much more complex. because the mastodons do not have synapses, right? So the cells can be handled to use... Yeah. And you know how they look, the cells? Because gene expression comes with an intermediate molecule, which is not a DNA, that is not a protein, but it is RNA. So RNA is the intermediate molecule between DNA and protein So DNA is always the same, but the identity is changing. So your identity in the DNA is completely different from the identity of your master. Even if the DNA is the same. Because identity is the way in which the gene is expressed. 

[MARK, 37.40]

OK, let's start with the building blocks. Nobody will ask you in this exam to draw what a nucleotide is, OK? It's just that you see it, and you don't get scared about it, OK? That is a nucleotide. The nucleotide has been depicted in different colors, red, yellow, and then you have some blue stuff, it is not depicted because it is very difficult, but here it is. So in this space, instead of this sediments, which is the nitrogenous base, there is the chemical formula of the nitrogenous bases. And we have one, two, three, four, five possible nitrogenous bases. These are called adenine, guanine, cyanosine, thymine, and uracil. So each building block that looks so small, and actually is very small. So what you do that is roughly one nanometer.

So very small, right? But it contains a lot of different things. These are called phosphates, this is called sugar, and these are called the nitrogenous bases. And all these parts do get organized differently in a DNA molecule. How is that? Here is the process of nucleotide polymerization. So basically you have one nucleotide that is addition to another nucleotide and a chemical bond is formed. So here is the first nucleotide, you have a reaction, a chemical reaction of a nucleotide with another one and you have a release of two pieces. two phosphates, which is not important. But basically, you have two nucleotides that are put together by a chemical bond. And it is called denucleotide, because it's done by two nucleotides. But again, you have a possibility that other nucleotides are additional here. And so you have a possibility that the chain is elongated, OK? It is linear, yes, it is, because one is after the other.

And whenever you have a chain done by less than 50 nucleotides, you talk about polynucleotide. Instead, if you have several nucleotides, much more than 50, then you can talk about polynucleotide. But still, in any case, you have linear polymers They are all linear changes. The chemical structure is very similar. If you have a look, you know, like that lady, find the differences. And you want to find the differences between these two. One, you realize that here there is an H. The H is an hydrogen. Instead, here you have an O H, which is an oxygen. That makes a very big difference between these two kinds of molecules. But it is one of the two differences they have from the rest. They are very, very similar molecules, OK? And then another thing that you see is that you have these bonds in between the nucleotides. And here they are. And you have here this letter code, A, T, G. for DNA, U, G, C for RNA. What are they? This letter code represents the nitrogenous bases that are here.

So whenever you see A, it means that in that position, there is another ring, there is this chemical structure. Whenever there is a G, it means that there is a one in this chemical structure. Whenever you see a C, Here there is a very big structure that is called adenine, here there is another structure called thymine and so on. It is called nitrogenous base because it is a chemical structure that contains nitrogen. And you see that nitrogen is an N in the chemical drawing. So it is a chemical structure that contains nitrogen, nitrogenous base. So why are nitrogenous bases so important? Now, the question is, are nitrogenous bases important? Yes, definitely important. Because they are the distinguishing feature of a nucleotide. is based on the different nitrogenous bases that each of the nucleotide has. So basically, the string that you read will always deal with the nitrogenous base present. OK?

This is why when you read a sequence of RNA or DNA, I don't know if you have ever seen it, usually a string of A, C, D, G, because that string corresponds to the nitrogenous base that is present on the nucleotide. Okay, so here is our real chemical drawing of the polymer, of the DNA polymer. You have the so-called backbone that is representing the linear chain of nucleotide addition one after the other one, okay? We have this chemical bond which is called phosphodiester bond, we don't care about it. However, it is a linear chain. And then you have all these nitrogenous bases being exposed. And is this important? Yes, it is important. Because this exposure allows the DNA chain with the possibility to search for another DNA chain, nucleotide chain, okay? And the search will always be like this, that the C present in one string will always search for a gene. And the A will always search for a T. Because these nitrogenous bases are able to create bonds one with each other. OK? And if this, of course, then what is important is that two poly-nucleotide chains come together. And they are held together by the interactions between the nitrogenous bases. So imagine a structure is formed in which you have the backbone of the sugars of the nucleotides that are in the external part, while in the internal part you have the nitrogenous bases interacting with each other. And when this occurs, it happens because you will have only always a G interacting with a C and a T interacting with an A nitrogenous base. So this is an example of secondary structure of double helix. How many chains do I have in a double helix? Two. But these two chains form one DNA molecule.

So, the two strands of, this is something I didn't say, okay? It's a difficult, it's a difficult thing, I won't explain it. It concerns the direction of the two strands, because if you have a look on, the direction of the chain that actually have been inverted by the pentagon of the sugar you will see that the pentagon here is pointing up and instead in the other sequence it will point down they will always pair like this because otherwise they don't pair they are called, sorry, antiparallel, because they go in two opposite directions, but this I think is too difficult a concept to understand, I don't ask you to learn it, okay? It's written down in the slides, just have a look, and if it's difficult for you, for your knowledge, perhaps you just forget about it. But what's important now to understand is that While you have a number of units of DNA form, the information that is inside this chain is the same of the other chain. Okay?

Because there is always the same plating between the ecological spaces. So, if you have a T here, you will know for certain that on the other side you will have a male, because there is no way that another nucleotide can be present in that position. So basically, when you have one sequence, you will always have also the other one. And in fact, the other one is called complementary, because they are the same thing. So when you have here T, you would always have a name on the other side. When you have here a G, you will always have a C on the other side. Right? OK. So this complementarity is also referred to as hybridization. So when you have a molecule of DNA hybridizing to another molecule of another chain of DNA, the two molecules must be complementary. So they have heterogeneous bases that are pairing one with each other. This concept is called hybridization and it is a distinguishing feature of nucleic acids. So why is that? Yes, please.

When you say that it is redundant, it is because I can infer the other chain from it. But the genes on the other chain cause something different. OK, big question. So the gene is the entire molecule. So the gene contains one strand and contains So the gene is a dominant. The slow information can be in one strand or in the other one. It depends on how it gets transcribed in tonality and then localization. But the gene itself is always doing this with the two strands. And the two friends do contain the same information. So how that information is stored, if using one strap on the opposite hand, depends. But the information you need is always obvious. It is a document.  Let's go. So, yes, this redundancy in the two nucleotides is a gift of evolution. Nature wanted a stable molecule, wanted a safe molecule, and that's in DNA form. DNA is very stable.

If I forget a DNA sample in the bench here, it won't get the right amount. I would say it will stay there. It is very stable. It doesn't change that easily. And basically, And it is a lockdown, so basically if I by chance lose the information that I have here, it will be always possible to recover it from the complementary strand. 


And what happens then is that the genes, genes organized in this DNA double helix, the nearly double axis held together by the interactions that are called hydrogen bonds between complementary neutron spaces and then it folds in three dimensions and the shape that it has really looks like this and of course you would have a very long double helix in our but then this double helix is very much compacted in order to enter the nuclei of our cells. 

[MARK , 54.17]

But the information to build up the DNA is in the primer sequence. And the primer sequence is the chain formed by the different nucleotides.

are all very similar to them, one with each other, but for the problem of trace. So, basically, it's not surprising then if a way to write down DNA sequence, and this way is called faster format, a text-based format for representing the DNA sequence, you have the nucleus are represented using single letter codes corresponding to the nitrogen phases, okay? So basically, this is an example of a fast format of a gene map. You have a first string that represents the header in which you have some information about the kind of DNA molecule that you are seeing. A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, C, D, E, A, B, Because for each nitrogenous base here, you would have the complementary one on the other side.

So actually, the exact way to write down the linear sequence would be this one with one chain and the complementary other. Is that OK? The point is that it's not necessary to write down the complementary one, because I know already the sequence. Because when I have an A, then I have a T. When I have a T, then I have an A. When I have a T, then I have a C, and so on. In DNA, I have four nitrogenous bases. A complementary to T, and G complementary to C. OK? That's it. So it's easier to store the sequence in using one of the two fronts. So I have put some So I put here a link, a web link, where you can also go. It is a link to the NCBI database, which is a database of DNA age. It's an American database, very, very much used in which all the DNA sequences of our genes are deposited. So if you are curious, to search some sequence of DNA or RNA or also proteins. You can do it through the NCBI. It's very common for us.

For example, here you have several different tools that allow you to navigate through different sequences. For example, here you have protein sequences. But now we're going to search for the gene. So I want to have a look to Jesus. And I got down here. I want to do surgery for the gene coding for human instincts. Yeah, it's used to regulate, by our body to regulate glucose levels and of course diabetes. especially the outer wound one, so type 1 diabetes, your cells in the pancreas are not able to produce insulin. So you have to take it as a problem. But insulin is a hormone that is produced by our pancreas physiologically, right? And the gene coding for insulin is here. It's a wonderful gene, a very easy one, by the way. It's one of the easiest genes we have. So in this database, we have several information. So you have all the codes. You see the name of the gene, the symbol. Each of our genes will have a symbol.

The species is human and it is a gene that encodes for monocromy because it's really easy for our genes. Here it is. And that is the gene. At some point you also have a sequence. Where are you sequence? Okay. So, I find the faster format of the gene, okay? So you have a letter code, I understand the gene is a kind of 450, something like that, Google very long, okay? This is the information necessary for our body to build insulin. OK? When I look at it, I always have to remember that actually the DNA molecule is not a single chain, but double chain. Double chain. And the two chains are complementary. So if I ask you to write down the complementary sequence to this one, you're able to do it. Even if you don't know the chemistry of DNA, right? Okay. Let's continue. So we now know that DNA is a double helix, but we know that the two parts of this double helix are redundant one to the other. They couldn't be the same information, okay? So this has been done on portage, because in this way, it is easier to replicate it. So is DNA replicable? Yes. Why it is replicated?

It is replicated to preserve genetic information across the cell generation. Think about development, the development in genome. You have a very, very small embryo that becomes an embryo. And that's also because of cell division. Also differentiation, but first of all, equation. the zygote that then becomes two cells, four cells, and so on, many cells, billions of cells, okay? And all the cells do have the same DNA. It means that every time the cell divides, before dividing, it duplicates DNA. So the scientists have been puzzled by how come the DNA can be duplicated by a cell. And in the years, they have proposed a lot of different hypotheses. And for example, a possibility is called distributional possibility. It's that when you have a mother DNA, then the content of the mother DNA is randomly assigned to the two daughter DNA molecules. And that was a possibility, but it turned out to be basically wrong. Another possibility is the conservative possibility.

So basically you have a double helix, and that double helix completely, entirely duplicates and becomes another identical double helix, the conservative one. That could be a possibility, but it was demonstrated to be basically false. And the only possibility that worked out, and it was experimentally demonstrated in the lab, is the semi-conservative mechanism. So basically, what happens is that the DNA unwinds. So you have the two strands wrapped one with each other. They unwrap. They get unwind. So you have same information used as a template to build up new strategy. And this is what is happening. So starting from one mother DNA will land in the tube. And in this tube, one chain will be the mother one. And the other one that is reported in the loop will be the daughter one, the new one. Well, because the first strand, the mother strand, is used as a template. It gives me the information to build up the complementary sequence.

So I have a mother strand here, and I have an enzyme, this enzyme is called DNA polymerase, that simply adds to a growing chain new building blocks based on the information that I have here. So if I have a T, then I will add an A, if I have a G, then I will add a C, and so on. So I have another strand, another chain, an enzyme uses as a template to build a complementary strand. That's it. The enzymes that support this activity is called DNA polymerase. And it is here represented as a hand, as a right hand. And indeed, analysis, structural analysis that were able to figure out the exact structure of the protein. Indeed, so that this enzyme, this protein, has the shape of a right-handed hand that is able to take a DNA molecule and that is able to slide on the D molecule using these parts of protein that are called thumb and fingers. And it does like this. So you have the DNA polymerase. You have one strand of DNA.

Basically, with the thumb, the enzyme is able to put the complementary nucleotide and then slide on the DNA to synthesize the complementary sequence. So what does the polymerase need? So you have a mother DNA sequence, right? I have a DNA polymerase. What does the polymerase need to synthesize with your strength? Well, it needs substrates. It needs building blocks. Right here is, again, the structure of the building block, the nucleotide, the sugar, the phosphates, and the nitrogenous base. You see, we never write that. The formula of the reduction space is just A, G, C, or T. So you can have four different nucleotides, four different building blocks that can be used by DNA polymerase. So you have mother strand of DNA, DNA polymerase. DNA polymerase will always work if you have a small sequence of a great acid that is hybridized to the DNA that you want to synthesize. So you have a strand of DNA, here it is. You have the DNA polymerase, you have the reading blocks.

that is hybridized to the molecule, and this structure is recognized by the DNA polymerase that gets attached, and when it's attached, then it starts to add new nucleotides. This sequence is called So that polymerization reaction only works if you have a primer. So if you have a single-strand DNA, a single-chain DNA, the DNA polymerase doesn't work. You always need a primer. then the chimer forms a double, a small double helix, and this structure is recognized by DNA polymerase, and you have polymerization. Why is that so important? Because I saw you, I mean, in a way, puzzled. Why is that so important? Because when you sequence DNA, and we'll come to that, you want to know the sequence of the genome, you use polymerization reactions. And in order to use polymerization reactions, you must have primers. Otherwise, DNA coordinates won't work and you won't get the sequence. So that's my need of telling you what a primer is.

So if you want to polymerize DNA, if you want to replicate DNA, you can do it, but you always need a primer hybridized, right? So you need a primer hybridized, then you need the nucleotides and the polymers. If you have these three reagents in one lab, you can replicate DNA. So here is just a picture of the logistics of DNA replication. So you have another DNA, here it is, that is unwinded, and each of these two filaments, represented in gray, can be replicated. By the DNA polymerase, you have one DNA polymerase for one strand. And the other DNA molecule is in the other strand. And then you have synthesis of DNA on both strands. So you have one mother, DNA molecules, and you end with the two daughters, DNA molecules. OK? Difficult. Kind of. The only thing I need to tell you is that I told you so far that the double helix is very stable, right? It is the molecule of radiating. If I forget it here, nothing will happen for months, right?

So is that so easy to break up another DNA molecule? No, no. This process requires energy, right? So, imagine, in your cells, whenever DNA verification starts, you will have a DNA molecule that is unwinded, and the two strands get separated. to do it in the lab, there is a very easy thing that I can do. I heat. So I give heat. I reach 90 degrees and the two strands get separated. Okay. When a double helix is closed, I'm talking about double strand DNA. When the duct linux is open, like in this configuration here, then we talk about single-stranded DNA. But two single-stranded DNA will always search one for the other because they are complementary and because they are more stable if they stay together. So if after heating, I cool down, these two strands will become double-stranded, okay? This process is called denaturation. So I can measure a molecule of DNA to open it, and then when it is opened, the two strands can be either used as tangles 

or they can close one or the other one, depends. Okay, another thing I need to tell you is that this magic enzyme, which is called

the D-L-U-M-I-N-E

enzyme, has a proofreading activity. What does it mean? that sometimes it makes errors. Is an error, grabe,

is an

error, a

replication

error something to be scared of? Yes or no? Yes, why?

Because the cell, some cell will have a predated

DNA

in some form.

I

think that's also the lead for tumors, but

I

don't know. Yeah. So the word is completely correct, a mutation. So a replication error by DNA polymerase is something serious. because a change in the nucleotide means a change in the genetic order. And this change can be silent if you're lucky, or can

lead to a fall if you are unlucky. Of course, or due to a condition of gene mutation, for example,

of rare diseases. and, for example, diseases of neuroticism with respect to the disorder, or for genetic mutations. So, replication errors are a serious problem. And the cell has several machineries to counteract replication errors. And the first machinery is the DNA polymerase itself. Basically, what happens is that if the DNA polymerase incorporates, by chance, the wrong nucleotide, So a nucleotide that doesn't match with the template, it stops in the sliding on the DNA, it stops. And it shifts the DNA in

another

position, this is called exonuclease position, in which there is another activity, It's not a polymerization activity, but it's a cut activity. Basically, what happens is that this is the DNA that is synthesized. This is the wrong nucleotide incorporated.

Well, the wrong

nucleotide incorporated is shifted by the tumor, and it's moved to another position to cut out the wrong nucleotide.

Okay?

This activity is called two-free activity. And what happens is that linicolimerase inserts one wrong nucleotide every 10 to the power of 5 nucleotides. But this rate is high, would be really high, and luckily, Thanks to the proofreading activity that it has evoked, this rate has been reduced to 1 to 10 to the power of 7. However, if we have a look to the mutation rate that we have for each cell division, this mutation, the genome is 1 to 10 to the power of 10. So basically what I want to tell you is that besides the proofreading activity, there are other several repair mechanisms that try to contain the number of mutations that are introduced for each cell division. Because a mutation is a problem. A mutation is a problem physiologically and is a problem also in the lab. If I want

to

If I want to detect a sequence of DNA, and I do it through a polymerization, and the polymerization produces a mutation, then I will get a wrong sequence. And the proofreading activity allows to contain the number of mutations that I can have. So here is, in this picture, the example of annihilation. Here is a mismatch, you see. The polymerase-by-chance has introduced the wrong nucleotide. It is not complementary to penetrate. It is called a mismatch. If it is not corrected, then in the second nucleation cycle, it will be considered as a normal nucleotide, and it will give rise to a different sequence in the headaches. So amputation is a permanent change in the DNA sequence. It can affect a single nucleotide, but also large genomic regions. So I told you the amputation rate, which is 1 to 10 to the power of 10, But this mutation is for each cell, but another reason is done by several cells. So basically, in one human carrier, you will have each generation 60 to 100 mutations.

These mutations can be single mismatches, they can be larger portions mutated, depends, sometimes some pieces of chromosomes are completely eliminated, or sometimes you have an extra chromosome, you know, Down syndrome for example, it is a pheasant of one chromosome, 21 from now on, and this occurs due to mutations, that is not contained by our physiological repair mechanisms. Another thing I need to tell you is that a mutation is something different from a polymorphism. So what is a polymorphism? A polymorphism is again a change in our DNA sequence. And the difference between a mutation and a polymorphism is in the frequency. So

mutations are very

rare.

Their

frequency is

less than

1%, while the polymorphisms have a higher frequency, higher than 1%. And it is not occurring for all genes. But there are some genes whose sequence is highly polymorphic. It means that the DNA polymerase incorporates errors during the replication of those areas and during generations. However, these errors are well-tolerated because the change in the sequence does not become a pathology. They are silent in most cases. They are neutral or vague. So in these cases you have a polymorphism, some change in your nucleotide sequence of your genes, which is in any case well tolerated. Instead a mutation is rare and often it correlates with pathology.

At the origin, it's always a mutation,

right? Yes.

At the

origin, it's always a mutation.

Yeah.

It is classified based on what brings after. What? It is classified by if it's good or bad after.

Based on the frequency that you have in the population, yes. There are some genes whose sequence becomes highly variable. And this mutation is well-collaborated. And if it is well-collaborated, evolutionally, it's not a problem. So you would have several individuals displaying different parts of genes with almost no phenotype. But there are some genes in which one single mutation can dramatically change the behavior and the phenotype, and can become a pathology,

for example. Okay? For example, cystic fibrosis, we know, is a pathology, a rare

pathology, and that is one thing that can change. Another is sickle cell disease, we know, and again, I should warn

you.

Another example, and we should have one look at the change, dramatically changing the fate of the holographic world status. So, it depends on the gene. And of course, if a gene is polymorphic, what happens is that the mutations are not generated because they do not end with a clear

pathological phenotype.

Okay.

Let's switch to the technological part. The technological part concerns the possibility that a polymerization reaction, we have already seen what a polymerization reaction is, how it works. We don't know everything about, right? But we know that you need

an

enzyme, which is called DNA polymerase. That is a very smart enzyme. That is polymerizing, but also correcting there. So we know something now. We know that it needs a fiber. We know that it needs nucleotides. OK. What I tell you now is that this mechanism, which is used by our cells throughout our life has been also exploited in the labs by molecular biologists to identify sequences. So if we now, in 2025, know the entire sequence of our genome, so if we know the order sequence of nucleotides of all our chromosomes, It is because we have polymerized

in the

lab our DNA. Okay? So we have taken samples of our DNA and we have performed polymerization reactions allowing to identify sequences. Okay? So the very, very important technique that

was

discovered In order to enable DNA sequencing through polymerization is the PCR. PCR stands for Polymerase Chain Reaction. It was invented by Harry Moolis. So Harry Moolis was a great scientist. He was awarded the Nobel Prize in Chemistry for his discovery. He was, you know, those kinds of people who turned out crazy scientists, that he was definitely a crazy scientist. He died a few years ago. And he wrote a book saying that without the use of LSD, probably he would never have his discovery achieved. So it was this kind of scientist, very, very striking, you know, the things he said. He also changed his wife seven times. I'm not judging him, right? I just want to share some information, because sometimes you think that a very good scientist always stays in the lab at work. He had a lot of fun, I think. And he made a very good story. By the way, his book is very great, I think, out of the box, definitely. So I recommend it.

So Harry Moolis tried and understood that there was a way in the lab to amplify DNA. What do I mean by amplify? I mean create, starting from one copy of DNA, create millions of copies of DNA equal to the original airplane. Okay? He did this discovery here. And

the

power of this discovery is that to do this, you just need these kind of machines that nowadays every biology lab in any structure, either a hospital or a research institute, will have, because it's a very cheap machine. It is a thermocycler. So basically it is a machine that contains several holes here. In each hole you make one reaction

with with DNA inside, so you can perform 100 reactions in the same time. And this is a thermocycler, it means that what the machine does, it changes temperature in time. Okay, so very easy machine. So, Kelly-Wood has discovered that if you put a molecule of the egg. And at the beginning, you increase the temperature 90 degrees. What does it happen? You already say that the eggs will open, right? Then in the same mixture, you have the primers. Here they are. You see? This is a primer. One primer is attaching to one filament, and the other primer is attaching to other filament, right? The primer will attach because you lower the temperature. So the primer gets attached. The double helix does not close because the temperature is not cold enough. So they are still open, but the primers get attached. And then you add the nucleotides and polymerase. And you change a little

bit.

Again, temperature so that the polymerase can elongate. You will end up with two molecules of DNA instead of one because you perform a polymerization reaction. Then, what the permacycle does, it repeats this reaction 30 times in the same tube. For each cycle, you will have the newly formed DNA will get opened. So now, at the beginning, you have one molecule of DNA open and synthesized. Then you have two molecules that open and get synthesized. So you end with four. But then you have another cycle in which you open, you touch diamond, and you elongate. So you will have eight. you have 60, you will have 32, and so on. Exponential increase of DNA molecules, okay? So basically, this PCR technique has become a team tool to amplify a DNA sequence, to identify, for example, the genome of viruses. Have you ever done a swab during COVID-19? Swab? Okay, so that tampone became a PCR reaction, right? In which the genome, the DNA of the virus was investigated.

If you have a viral genome, then the virus will get attached to them, and you will have PCR and DNA amplification. No virus, no DNA amplification. You get it? I repeat it. OK, too much, sorry. But some of you got it, right? I repeat it. You do a swab. You are full of COVID-19. So you have coughing, fever. And you have virus there. And the virus has a genome. A genome is, as per human, the DNA sequence that encodes for the virus, okay? But the genome of the virus is known because it has been sequenced. So basically, I will always be able to design primers for that. Because if I know a sequence, I can design a DNA molecule that will attach to it because it is complementary, right? So imagine you have your swab. Your swab is full of viral DNA. I add polymerase, primers recognizing the virus genome, and nucleotides. And then I put it in the thermal cycle. After 30 cycles, if the virus is there, I will have a lot of copies of viral genome. If the virus is not there, I will not have

amplification. If you have COVID-19, then

you have a genome that is known, and I can design the primers for that. I can predict the sequence and design primers for that.

Great question. You

could have been in molecular biology. So you need as few as 16 nucleotides to have specificity of recognition. So in the human genome, the possibility that you find the same 16 How is it going? Okay. I think I'm almost ready. I can go on another 10 minutes if you allow me. Can I go? Because I'm a bit late. Because it took me a little bit long. Because typically my students are biologists and not informaticians. It was an experiment, I told you. Okay, so just two slides of a flavor of how PCR works. You heat 94 degrees with a thermocycler, 1,000 euro in our lab, the cheapest instrument possible in our lab. If you don't have money to buy a PCR, then you're very, very poor, because it's one of the cheapest machines we have in the lab. So double-stranded DNA handling, you heat it, 94 degrees, then you lower a little bit the temperature. So you switch the temperature a little bit. This is not enough to let all the double helix to close, but it is enough to attach the divers.

And then you will have synthesis of DNA with the DNA polymerase, because the polymerase recognize the hydrolyzation as nucleotides. And you will have the polymerization. So you have one molecule, you end with two. If you repeat this cycle for 30 times, then you will have two to the power of 30 molecules, ideally. then practically this is not happening because in the reaction you have a limited amount of nucleotides that get consumed, little amounts of primers that get consumed, and enzymes that get tired, right? So, but ideally, you can have two to the power of 30, two to the power of cycles, right? Okay. So again, phases of a PCR cycle, denaturation, implantization, and extension. Can

you have some mutation during this process?

Of course, yes. Of course, yes. You can have a mutation because you are using polymerase, and polymerase fails sometimes. They do have proofreading activity, but still they introduce mutation. And here we get to a very important point. Once you want to sequence a DNA stem, in the sequencing techniques, you amplify sequences by PCR before sequencing. Because in this way, you will have a lot of repetitions of the same sequence. Because if you repeat a sequence of a stretch of DNA, you will always be able to recognize errors. Because you will have the same sequence sequenced, let's say, 20 times. And in 20 times, the possibility that you have the same error will be low. So you will always be able to recognize a sequence error, a polarization

error. OK? So you take something like a common different class and you get the code

here? Yes, exactly. OK. So PCR, and yes, this is the last slide.

What was

number

23 again?

So PCR can be used for sequencing. And so we started with a physiological mechanism, DNA replication. Then we switched this mechanism in the lab and we invented the PCR. And then we realized that PCR can be used for DNA sequencing. So I'm making a very long story short now. It was 2000 and the human genome project was ended. The complete sequence of the human genome was sequenced and it was a huge effort done of several in order to obtain the sequence of all our chromosomes. And they were done through polymerization and through PCR reactions. DNA sequencing through polymerization and through PCR has evolved the technologies over the years. I told you that 25 years ago, the genome was sequenced for the first time in the Human Genome Project, right? And that sequencing technology at that time was called Sander sequencing. And it is the first generation of sequencing technologies. How many generations do we have? Three.

So in 25 years, there was a dramatic change in the sequencing technologies. And after the Sanger sequencing, the first generation sequence, the second generation came, and now we are at the third generation sequence. The technologies are different in each case. I will try my very best. to reach a third generation sequencing tomorrow. But we'll see. I'm not sure yet, OK? So here is how the sample sequencing works. Suppose you have the sequence that you want to determine here. You have to denature it to make a single strand. Here it is. Then this sequence of interest you want to sequence must be preceded by a sequence that you know because in this way you are able to design a primer on top of it. Right? Here it is. So you have this scenario. You have a molecule of interest you want to sequence. You have a primer on top of it. And then you have polymerase, because without polymerase it's not working. And you have building blocks. But in this case, you use special building blocks.

These special building blocks are called B-deoxynucleotides. This name is very difficult, it's not important. But the important thing is that these nucleotides lack the possibility to elongate a chain. So, they are called chain terminators. Once they are incorporated, the chain won't elongate anymore. Okay? So, now think that you have your sequence of inputs, and you have these terminator nucleotides. For example, here you have a terminator with

A.

So basically, the aim will be incorporated any time in your sequence there is a peak. Right? Basically, if I have three peaks, I will have three fragments terminated in each peak position. Right? So now imagine to do this reaction with all four primes. of terminator nucleotides, you will end with several fragments synthesized, each terminated in one specific position. So if you now analyze the newly synthesized fragments, they all differ in length, and you are able to identify the sequence, starting from the smoothest ending to the longest. So if the first, the smallest fragment was done by introducing an A, it means that in the original sequence there was a T. And so on. If the second fragment was obtained by addition of a G terminator, it means that in the template there was a C. And this allows you to identify the sequence. How long can I push a Sanger reaction? So the Sanger reaction is very efficient. I can push it until 1,000 nucleotides, which is actually a lot, but it's not enough. I can push a Sanger sequencing throughout a chromosome, because a chromosome is millions of nucleotides long. How did I solve this problem? By fragmenting the DNA. So DNA was fragmented so that my reaction could work.

And then I obtained fragments through which I had to reconstruct my genome informatically. Okay? I stopped.