
So yesterday we talked here. Is there the English speaking? Yes, okay. So yesterday we talked, we introduced structural and functional aspects of DNA, focusing on DNA replication. Because this mechanism is the instrument of polystyrene the role of DNA as a molecule of a hierarchy, but it's also instrumental to understand the basics of the technologies that are used for DNA sequencing. And technologies used for DNA sequencing are technologies underneath the bioinformatics analysis of genomics and transcriptomics, right? So yesterday we stopped here. We talked about DNA replication. Then we switched to the PCR. We introduced the reaction of PCR, which is a reaction that basically exploits DNA polymerization to amplify a sequence. And yesterday we ended saying that you can change a little bit, modify a PCR reaction to make it work as a sequencing reagent. And this has been changed through generations of technologies. So we have three generation of sequencing technologies.

The first generation of technologies is the sound wave sequencing. So the sound wave sequencing is the technology that has been used to sequence the human genome for the first time. It was a huge project that costed a lot of money, a lot means that I don't know, but that they are really a lot. And the project took, let's say, five to 10 years, okay? So today, with our generations of sequencing, and it was 25 years ago, so today, with the knowledge we have now in our hands, sequencing of a human genome costs I don't know the exact number, but thousands. If I want to sequence today, one piece of meat, how much would it be? A piece of meat, let's say as long as thousands of times. The machine is not the same of last time, of the human genome. It is indeed, for the short pieces of DNA we use, we still use some sequencing nowadays. So one piece of, one string of DNA, three, right?

The point is that if you perform a Sanger and you push it for the whole genome, then it's a million nucleotides. That is a big economic problem. But for this, we did the shift of technology. And we did the shift to a second generation of technology. It is called Illumina technology. But now, let's go back to the Sanger. So with the Sanger sequencing, basically, you exploit the fact that you can polymerize DNA strand, here it is. If you have a primer, of course, because the primer is needed for the DNA polymerase. Then after the primer, you have DNA polymerase. And after the DNA polymerase, you have particular building blocks. So these building blocks are called chain terminators. And basically, when they are incorporated, the reaction stops. Because the nucleotide can't be eliminated here anymore. So assuming that you are in a reaction, so you have excess of these three agents, and you have the four of them, imagine that they are randomly incorporated almost in every position.

So basically, at the end of an amplification cycle of a PCR reaction, with a sample of fragments of DNA that are then linked to that position. So by measuring the length and knowing, of course, which nucleotide has been used to terminate that sequence there, you can infer the sequence. That's it. And that's the Sanger sequence. So the throughput of this technology has been And this throughput was absolutely useful for the sequencing of the new genome, right? So in this, we have a lot of unforeseen configuration. What happens is that a algorithm is used to identify each nucleotide, each terminator. So, as you see here, you have the telometer in a color of yellow, for example, the telometer in a color of blue, in green, and in red, and so on. So basically, after the polymerization reaction, you end with fragments that not only differ from their length, but they also differ from the color.

So when I say color, there are several things of chemistry and physical chemistry that I have no time to explain and that you don't know. But imagine this coil as a sort of mini-mini lamp that can be attached to a nucleotide. So here you have the chemistry of the nucleotide. Imagine that here you have a small structure that needs light. of a particular color. That's a color for milk. It's called chloroform. But who cares now? Here we have all these fragments terminated in different colors. So what happens now is that we use light to excite these lungs, like we use with tongue. each of these lamps, and then you record the color emitted, right? So yellow lamp, green lamp, blue lamp, and red lamp. Switch on and record the light. The light is recorded and converted. So the light is a number of photons. You know what photon is? Yeah. It's like a particle of light. Yeah, so you recover a number of photons for each color.

And this number of photons, of course, can be converted to a graph, like a peak in a graph. So imagine here you have on the y-axis, you have the intensity number of photons, right? So basically, you measure And of course, you have a means for the fragments on the land. And that is basically done. So to show us the fragment, you have a green peak. A green peak is a sea. And then you have a red peak. A red peak is a sea. And so on. Using this technology and using this barcode label of nucleotides, perinatal nucleotides, basically the development of the Sanger sequencing was achieved. And let's talk a little bit, as you will do, about another important concept now, which is referred to the precision of sequencing. We showed you yesterday that the DNA polymerase does make errors. from time to time. So an error of DNA polymerase in the human genome, in our cell, is a mutation. An error in the sample sequencing by DNA polymerase is a sequencing error.

It means that I recovered a wrong sequence, right? So I have to have means to avoid this, to increase the precision of sequencing, to overcome the mutations of the enzymatic activity of DNA polymerase itself. So this can be done in one way, increasing the number of cycles. Because if I increase the number of cycles, then I accumulate several fragments. Even fragments of the city land are generated several times. So I compare them, I can align them, and I do see Another possibility is this one. Always remember, we said it yesterday, there are DNA strands in the donor. It means that the same information is contained in two strands, right? So I can always sequence one strand and I can also sequence the other strand, okay? What I get in here So sequencing both strands is a way to increase the precision of sequencing because the sequence is always the same.

I guess the second method is better than the first one, because in the first one you have like, for the first part of the sequence you have multiple, many copies, while for the last, for the most part, you have at least and if you do reverse you have, you balance more and you reduce errors more, I guess. Yes, that's a good guess. So molecular biology is absolutely the way of thinking of bioinformaticians. I discovered it yesterday and today. OK, so now in the end, you end with a lot of different fragments. Here we are, for example. And what you see, you can see immediately. So there should be a gray rectangle here. It's visible here, for example. And it's visible in your slides. So basically, you do see immediately that in the read-for, for example, there is an error. Because here, the important reason to incorporate a T while you know the other family has been incorporated.

So this kind of procedure, which is called alignment, and it will be the focus, I think, of your next lectures by Silvia. So it's a procedure that allows you to reveal a sequence starting from the second replicates of the same sequence, okay? And, of course, these replicas allow to increase sequencing procedure. But another issue is that, and you realize it immediately from here, is that these sequences are all similar but not identical. And they share some bases. For example, this and this share three nucleotides, and while this and this are large nucleotides. And this is also very useful, because typically the length of a lead is much less long than the entire sequence that I want to sequence. So basically, if I have a set of documents and they are all partially overlapping, then what I need to do is retrieve the entire sequence from the assembly of the different fragments, right? And this process is called sequence assembly.

So sequence assembly was an issue during Samuel sequencing being used in the sequencing of the human genome for the first time. It was an issue, right? Because some of the sequencing sequences, as long as one finds the nucleotides, then basically the reaction doesn't work anymore. However, assembly has become the issue of the next generation sequencing technologies. Because in the next generation sequencing technologies that we're going to see, now let's see now that we're going to see now the fragments they are a little shorter so typically they don't reach 100 maximum 300 and then you have the pieces that are 100 long these layers and you have to reconstruct the entire human genome so gene assembly And you see, I guess, a lot of different ways to do it. Okay, just spoiler, some challenges during this reconstruction of planets. The human genome is tricky. There are, we see that, pieces of DNA that are just repeating. you have almost identical repeat sequences and they are very normal.

They are repeated throughout a different position throughout the genome. So now imagine you got a read for this part of the genome and another read for this part of the genome. What can happen is that you realize that there is an overlap between these two reads

